### PR TITLE
feat(simplifiedRow): adding new methods in dotnet

### PR DESCRIPTION
**Pull request by AkankshaAcharya**
These changes include 2 new methods ConvertToDynamicObject() and ConvertToTransposedDynamicObject(), which are applicable only for the simplified row stach format.

### CODE REVIEW

**Review by Ramanth — APPROVED** [review]
Looking good. left minor comment

File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/ISimplifiedRowStachExtension.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace FactSet.Protobuf.Stach.Extensions
+{
+    public interface ISimplifiedRowStachExtension : IStachExtension
+    {
+        List<dynamic> ConvertToDynamicObject();
```
Comment (saigiridhar21) [inline dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/ISimplifiedRowStachExtension.cs:7]: Is it better to call this method `ConvertToDynamicObjects`? Also, can you add some documentation for this method?

File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/SimplifiedRowOrganizedStachExtension.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                        : columnDefinition.Description;
+                    headerRow.Add(description);
+                }
+
+                foreach (var dataRow in table.Data.Rows)
+                {
+                    var data = new ExpandoObject() as IDictionary<string, dynamic>;
+                    for (var i = 0; i < dataRow.Values.Fields.Count; i++)
+                    {
+                        data.Add(headerRow[i], StachUtilities.ValueToObject(dataRow.Values.Fields["col_"+i]));
```
Comment (saigiridhar21) [inline dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/SimplifiedRowOrganizedStachExtension.cs:38]: Is it better to create a constant for `col_` in Stach Utilities?

File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/StachUtilities.cs [hunk 3]
```diff
@@ -34,6 +39,38 @@ public static string ValueToString(Value value)
             }
         }
 
+        /// <summary>
+        /// Returns the actual value object instead of the Protobuf Value object.
```
Comment (Ramanth) [inline dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/StachUtilities.cs:43]: Lets change this to "Returns the underlying raw object from the protobuf Value object"

**Review by Ramanth — APPROVED** [review]
Looking good. left minor comment

**Review by saigiridhar21 — APPROVED** [review]

**Pull request merged**